homeassistant/components/mqtt/__init__.py

```diff
@@ -89,6 +93,9 @@ def valid_publish_topic(value):
         vol.Optional(CONF_USERNAME): cv.string,
         vol.Optional(CONF_PASSWORD): cv.string,
         vol.Optional(CONF_CERTIFICATE): cv.isfile,
+        vol.Optional(CONF_CLIENT_KEY): cv.isfile,
```

Comment:
Since client_key and client_cert depend on one another, add that to the config validation using [`vol.Inclusive`](https://github.com/alecthomas/voluptuous/blob/1bda974756678a45ad800725829e23a1a398849c/voluptuous/schema_builder.py#L880)